### PR TITLE
as a funder, i want to advertise that i am hiring on my bounty, so bounty hunters can take this into account when attempting my bounty

### PR DESCRIPTION
**Pull request by owocki**
##### Description

as a funder, i want to advertise that i am hiring on my bounty, so bounty hunters can take this into account when attempting my bounty

screen_shot_2018-06-16_at_12_23_25_pm

[video of funder form](http://bits.owocki.com/0x1D0h0v1p1i/Screen%20Recording%202018-06-16%20at%2012.24%20PM.gif)

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

bounty

##### Testing

tested it

##### Refers/Fixes

self

### CODE REVIEW

**Comment by owocki** [comment]
we could also collect the following info:
* position type: contract, part time, full time
* compensation info (rate, equity, etc)
in a v2 we could also maybe use this information to populate a gitcoin job board

**Comment by owocki** [comment]
heres a content piece i wrote about this https://medium.com/gitcoin/hiring-with-gitcoin-just-got-better-259c59aa8e3

File: app/assets/v2/js/pages/bounty_details.js [hunk 1]
```diff
@@ -578,6 +578,8 @@ var do_actions = function(result) {
     var should_block_from_starting_work = !is_interested && result['project_type'] == 'traditional' && (result['status'] == 'started' || result['status'] == 'submitted');
     var show_start_stop_work = is_still_on_happy_path && !should_block_from_starting_work;
     var show_github_link = result['github_url'].substring(0, 4) == 'http';
+    var show_job_description = result['attached_job_description'] && result['attached_job_description'].substring(0, 4) == 'http';
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/bounty_details.js]: update this to `startsWith('http')` ? makes code more redable

File: app/assets/v2/js/pages/bounty_details.js [hunk 2]
```diff
@@ -692,6 +694,21 @@ var do_actions = function(result) {
 
       actions.push(_entry);
     }
+
+    if (show_job_description) {
+      var url = result['attached_job_description'];
+
+      var _entry = {
```
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply app/assets/v2/js/pages/bounty_details.js]: we need to rename this to `let` once https://github.com/gitcoinco/web/pull/1476 gets in

File: app/assets/v2/js/pages/bounty_details.js [hunk 3]
```diff
@@ -692,6 +694,21 @@ var do_actions = function(result) {
...
+        enabled: true,
+        href: url,
+        text: gettext('View Attached Job Description'),
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/bounty_details.js]: Should we rename this to something shorter? ( 'Recruiting`)

File: app/assets/v2/js/pages/new_bounty.js [hunk 4]
```diff
@@ -117,6 +117,18 @@ $(document).ready(function() {
   // denomination field
   $('select[name=deonomination]').select2();
 
+  $('#hiringRightNow').click(function(){
+    setTimeout(function(){
```
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply app/assets/v2/js/pages/new_bounty.js]: Do we need the timeout ?

File: app/dashboard/templates/submit_bounty.html [hunk 5]
```diff
@@ -137,7 +137,20 @@ <h4 class="mt-5">{% trans "Creator Details" %}</h4>
                 {% include 'shared/github_username.html' %}
                 {% include 'shared/notification_email.html' with show_information_publicly_checkbox=1 %}
                 {% include 'shared/name.html' %}
+                <div class="w-100 mt-2">
```
Comment (thelostone-mc) [inline app/dashboard/templates/submit_bounty.html]: Should look alright without `w-100`

File: app/dashboard/templates/submit_bounty.html [hunk 6]
```diff
@@ -137,7 +137,20 @@ <h4 class="mt-5">{% trans "Creator Details" %}</h4>
...
+                  <label class="form__label" for="hiringRightNow">{% trans "Hiring right now?" %}</label>
+                  <div class="hiring_container">
+                    <div class="form__checkbox">
+                      <input name='hiringRightNow' id='hiringRightNow' type="checkbox" value=1 />
+                        <label class="form__label" for=hiringRightNow>{% trans "I am hiring for a full time, part time, or contract position, right now and would like Gitcoin to attach this job description to the bounty." %}
+                      </label>
+                    </div>
+                  </div>
+                  <input name='jobDescription' id='jobDescription' type="url" class="w-100 form__input hidden" type="text" placeholder="https://link.to/job/description" value="" />
```
Comment (thelostone-mc) [inline app/dashboard/templates/submit_bounty.html]: Should default to 100% without `w-100`

File: app/assets/v2/js/pages/bounty_details.js (hunk 1 above)
Comment (owocki) [reply app/assets/v2/js/pages/bounty_details.js]: updated

File: app/dashboard/templates/submit_bounty.html (hunk 5 above)
Comment (owocki) [reply app/dashboard/templates/submit_bounty.html]: thx updated

File: app/assets/v2/js/pages/bounty_details.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (owocki) [reply app/assets/v2/js/pages/bounty_details.js]: done

File: app/dashboard/templates/submit_bounty.html (hunk 6 above)
Comment (owocki) [reply app/dashboard/templates/submit_bounty.html]: thx updated

**Comment by owocki** [comment]
shipping this now

> this would be complete once we get the jobs section in :')

agree!  we can use this as seed data for the jobs stuff

**Pull request merged**